### ISSUE TITLE
Detects serverless template as cfn template

### ISSUE THREAD

**Issue by alexjurkiewicz** [awslabs/aws-cfn-lint-visual-studio-code#43]
I created a new serverless.yml with this command: `npx sls create -t aws-nodejs -n my-service`, and cfn-lint is trying to lint it. Here's a minimal test case for a file that is detected as a valid file to lint:

```yaml
#  Resources:
#    NewResource:
#      Type: AWS::S3::Bucket

foo: bar
```
It seems like the commented code is triggering the detection somehow.

**Comment by kddejong** [comment]
Oh shoot.  Sorry about that.  Seems like an easy fix.  I'll try to get that in shortly.

**Comment by kddejong** [comment]
I think we have this fixed in 0.7.0 now.
